package.json

```diff
@@ -43,6 +43,11 @@
     "react-transition-group": "^4.4.1",
     "react-twitter-embed": "^3.0.3",
     "react-typography": "^0.16.19",
+    "rehype-react": "^7.0.3",
+    "rehype-sanitize": "^5.0.0",
+    "remark": "^14.0.1",
+    "remark-parse": "^10.0.0",
+    "remark-rehype": "^10.0.0",
```

Comment:
you'll want to use these versions to get around transpiling packagges:

"rehype-react": "^6.0.0",
    "rehype-sanitize": "^4.0.0",
    "relay-runtime": "^12.0.0",
    "remark": "^13.0.0",
    "remark-parse": "^9.0.0",
    "remark-rehype": "^8.0.0",
    
    https://github.com/gladly-team/tab-web/pull/242/files#diff-882b2c04b01b2e8b2cdcf1817c30ea503a8005f1c0d54cfff37053b6801fea85R4-R9